vtrace/envitools.py

```diff
@@ -4,15 +4,23 @@
 import envi
 import envi.memory as e_memory
 import envi.archs.i386 as e_i386  # FIXME This should NOT have to be here
+import envi.archs.i386.regs as eair
```

Comment:
Honestly, we might want to just merge them into this one and expand the entire lockstepper test.

Reply:
that makes sense.  i'm not sure how at the moment.  i don't like LockStepper being in the unittest, but i guess it makes sense on some level.  mine is likely faster (since it doesn't do exhaustive memory comparisons every instruction) but it's also less accurate (for the same reason).

Reply:
agreed.